sources/api/README.md

```diff
@@ -143,12 +143,15 @@ Some day we may need to make an earlier service (say, apiserver) configurable th
 This usage is scoped to the keys that have changed, updating relevant config files and restarting affected services.
 See [thar-be-settings](thar-be-settings/) docs.
 
-### configured.target
+## Boot stages
 
-This is a systemd target that depends on [settings-applier](#settings-applier) and represents the point at which the system is fully configured.
+Bottlerocket's boot sequence includes three systemd targets:
 
-Applications can depend on this in their service definition.
-Services like Kubernetes and containerd depend on this.
+- `preconfigured.target`: this is the target where all the aforementioned services run.
+Once all the dynamically generated configurations are created, the system transitions to the next target
+- `configured.target`: this is the target that represents the point at which the system is fully configured, any configured [bootstrap container](../../bootstrap-containers-settings) will run at this stage to apply further configurations in the system.
```

Comment:
Double use of the word "configured" seems confusing here. Unless there are two types of bootstrap containers (i.e. "configured bootstrap containers" and "some other type of bootstrap container") then I would recommend this:

```suggestion
- `configured.target`: this is the target that represents the point at which the system is fully configured.
   [Bootstrap containers](../../bootstrap-containers-settings) will run at this stage to apply further configurations in the system.
```